scripts/funcotator/data_sources/gnomAD/liftoverVcf.wdl

```diff
@@ -0,0 +1,373 @@
+#     Int  preemptible_attempts                         -  Number of times to allow each task in this workflow to be preempted.
+#     Int  disk_space_gb                                -  Amount of storage disk space (in Gb) to give to each machine running each task in this workflow.
+#     Int  cpu                                          -  Number of CPU cores to give to each machine running each task in this workflow.
+#     Int  boot_disk_size_gb                            -  Amount of boot disk space (in Gb) to give to each machine running each task in this workflow.
+#
+# This WDL needs to decide whether to use the ``gatk_jar`` or ``gatk_jar_override`` for the jar location.  As of cromwell-0.24,
+# this logic *must* go into each task.  Therefore, there is a lot of duplicated code.  This allows users to specify a jar file
+# independent of what is in the docker file.  See the README.md for more info.
+
+workflow LiftoverVcf {
+    String gatk_docker
+
+    Array[File] variant_vcfs
+    File chain_file
+    File target_reference_sequence_fasta_file
+    File target_reference_sequence_fasta_file_index
+    File target_reference_sequence_fasta_file_dict
+    String lifted_over_vcf_name
+    String lifted_over_rejects_vcf_name
+
+    Boolean? warn_on_missing_contig
+    Boolean? write_original_position
+
+    File? gatk4_jar_override
+    Int?  mem_gb
+    Int?  preemptible_attempts
+    Int?  disk_space_gb
+    Int?  cpu
+    Int?  boot_disk_size_gb
+
+    # Run liftover on each input VCF:
+    scatter ( vcf in variant_vcfs ) {
+      call LiftoverVcfTask {
+          input:
+              input_vcf_file                              = vcf,
+              chain_file                                  = chain_file,
+              target_reference_sequence_fasta_file        = target_reference_sequence_fasta_file,
+              target_reference_sequence_fasta_file_index  = target_reference_sequence_fasta_file_index,
+              target_reference_sequence_fasta_file_dict   = target_reference_sequence_fasta_file_dict,
+              lifted_over_vcf_name                        = lifted_over_vcf_name,
+              lifted_over_rejects_vcf_name                = lifted_over_rejects_vcf_name,
+
+              warn_on_missing_contig                      = warn_on_missing_contig,
+              write_original_position                     = write_original_position,
+
+              gatk_docker                                 = gatk_docker,
+              gatk_override                               = gatk4_jar_override,
+              mem                                         = mem_gb,
+              preemptible_attempts                        = preemptible_attempts,
+              disk_space_gb                               = disk_space_gb,
+              cpu                                         = cpu,
+              boot_disk_size_gb                           = boot_disk_size_gb
+      }
+
+      call IndexFeatureFileTask as IndexContigFiles {
```

Comment:
Is it okay that the mem, disk space, etc cannot be controlled on any task except `LiftoverVcfTask`?

Reply:
Oversight!

Fixed.